### DISCUSSION TITLE
my app with create react app dont works

### DISCUSSION THREAD

**Discussion by LucasDelgado** [vercel/vercel#5111] [Help]
My app dont works. And it has not anything, it is simple, just it includes reactRouter, but vercel throws errror. 
mi site is https://lana-75wei0z82.vercel.app/ . the repository is https://github.com/LucasDelgado/lana . 
can anybody help me? or any ideas....
Thanksss

**Comment by mcsdevv** [comment]
Hello Lucas,

Please share with us the full error so we can advise.
